### PR TITLE
DOC-2515 speedb changes

### PR DESCRIPTION
**Pull request by kaitlynmichael**
Jira: DOC-2515

### CODE REVIEW

File: content/kubernetes/re-clusters/auto-tiering.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ Below is an example REDB custom resource:
 apiVersion: app.redislabs.com/v1alpha1
 kind: RedisEnterpriseDatabase
 metadata:
-  name: rof-redb
+  name: autoteiring-redb
```
Comment (AlonMagrafta) [inline content/kubernetes/re-clusters/auto-tiering.md:76]: @bascar can you confirm this?

**Review by AlonMagrafta — COMMENTED** [review]
LGTM, I left @bascar one question to confirm.
@kaitlynmichael Thanks. If you also know the answer then we can proceed :)

File: content/kubernetes/re-clusters/auto-tiering.md (hunk 1 above)
Comment (bascar) [reply content/kubernetes/re-clusters/auto-tiering.md:76]: @AlonMagrafta @kaitlynmichael API is not changing, so this should not change.

Comment (bascar) [reply content/kubernetes/re-clusters/auto-tiering.md:76]: Actually, this is just a name, with no impact. Can stay this way, sorry.

**Pull request merged**